.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node
+sudo: required
+services:
+- docker
+env:
+  global:
+  - REGISTRY_USER=ocadotechnologygitlab
+  - IMAGE_NAME=ocadotechnology/statsd
+  - secure: YbNxrr1D1AkcqR6YGFvwQIkxD6X6fYt5qSr6Cc+yTVQgh81kaJEfz30iu39rfO3mb6kHvVnfxF2xOnF65gc67tShxvYQTMGwmRcmk5cDd+VKS23XPicvta/hrs90cIF6XQBAseoAqRGRyE8s70mELEb60GuwERvIkcJV/dppCYcIkf3H977OdtIwIAfWNTPoIVVhHID70Edjg33oJtoVu3NBeXZcJIJCw0vy649NBSrSgp2j+YZRxbaAUizSslVQJ1c6tYnP12dCzkHHVtC+mj5fWxwjTi//vwknckB4YFMeH7SVjU12VB6XYzjFYxrSLaWqCuwWjcDLrgx5RS0tAZQeD+sX5xCeXh8UenEsAvx5TyNLfuezlS3/+xZ6RKhtNOuWWT+/QgDHlpZ2s/LISqgchW2yWp0fbtnuz+WhLmSbtCTKVrR/++yDtk4QiiFJFG8WtI4903Cjg9A9INl3Zmiw9THuW8sHHJ6gGuaN8IPH9ydstWjx7Q62qDW+3nbEv6cpfF7MsVaD4IqYzj4JWv0ZU2ah9G5sk4WHFF4HtEHu3yO3EQuyShl8hYvPM/oG7bdGYASNBYkEBNEj/choaDD/1VV0jiCwegeWbxxVnSA8XsUiWsD5xs4KC6bh3lZPMqvzZpKxyh7NrAncjO0xjo2BCtzG8/E13uP6n/ZNXmg=
+before_script:
+- docker pull "$IMAGE_NAME" || true
+script:
+- docker build --pull --tag "$IMAGE_NAME" .
+after_script:
+- docker images
+before_deploy:
+- docker login -u "$REGISTRY_USER" -p "$REGISTRY_PASS"
+- docker tag "$IMAGE_NAME" "${IMAGE_NAME}:latest"
+- docker tag "$IMAGE_NAME" "${IMAGE_NAME}:${TRAVIS_COMMIT}"
```

Comment:
Should we do this only on tags? Use semantic versioning?

(And possibly worth not even making available `:latest`?)